package.json

```diff
@@ -2,10 +2,10 @@
   "name": "circleci-status",
   "displayName": "CircleCI Status",
   "description": "CircleCI Status is a VSCode extension to check CircleCI build status.",
-  "version": "1.0.5",
+  "version": "1.1.0",
   "publisher": "kuromoka",
   "license": "https://github.com/kuromoka/circleci-status/blob/master/LICENSE",
-  "homepage": "https://github.com/kuromoka/circleci-status/blob/master/README.md",
+  "homepage": "https://github.com/doyou89/circleci-status/blob/master/README.md",
```

Comment:
please leave it as it is.
https://github.com/kuromoka/circleci-status/blob/master/README.md

Reply:
back to the original homepage in 752b721 commit.